src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -259,6 +259,20 @@ Apply to thead to make headers not bold.
     padding-top: 0;
 }
 
+.question-recommended-length {
```

Comment:
Is there any better place to put these rules? Currently it seems to be in between some random rules